src/main/components/email/Integrations.jsx

```diff
@@ -2,21 +2,23 @@ import React from "react";
 import Content from "../common/content/Content";
 import CardComponent from "../../molecules/card/Card";
 
-const IntegrationComponent = () => {
+import { integrateGmail } from '../../../utils/integrationUtils';
+
+const IntegrationComponent = (props) => {
```

Comment:
Maybe we can change the name? displayEmailsComponent

Reply:
This component is not for displaying emails. Instead, it is to start the email integration with either Gmail or any other services out there.